setup.cfg

```diff
@@ -32,7 +32,7 @@ astropy_header = true
 text_file_format = rst
 xfail_strict = true
 remote_data_strict = true
-addopts = --doctest-rst
+#addopts = --doctest-rst
```

Comment:
Don't do this change, it's keep coming back, I suspect it maybe something outdated on the main brach of your fork?

Reply:
Sorry - Didn't mean to commit this change.